src/app/components/MediaLoader/index.tsx

```diff
@@ -148,28 +159,31 @@ type Props = {
 };
 
 const MediaLoader = ({ blocks, embedded, className }: Props) => {
-  const [isPlaceholder, setIsPlaceholder] = useState(true);
   const { lang, translations } = useContext(ServiceContext);
+  const { pageIdentifier } = useContext(EventTrackingContext);
   const { enabled: adsEnabled } = useToggle('ads');
 
   const {
     id,
     pageType,
-    counterName,
     statsDestination,
     service,
     isAmp,
     isLite,
     showAdsBasedOnLocation,
   } = useContext(RequestContext);
 
+  const showPlaceholder = !PAGETYPES_IGNORE_PLACEHOLDER.includes(pageType);
+
+  const [isPlaceholder, setIsPlaceholder] = useState(showPlaceholder);
+
   if (isLite) return null;
 
   const producer = getProducerFromServiceName(service);
   const config = buildConfig({
     id,
     blocks,
-    counterName,
+    counterName: pageIdentifier,
```

Comment:
cc @alex-magana and @holchris - we will need to slightly change our counterName override changes, because we've realised the counter name should come from the EventTrackingContext instead of the Request Context. In our MediaLoader for On Demand TV PR, we need to override the page identifier instead, and also remove the setting of the countername on the RequestContext on the PageWrapper